go-controller/pkg/ovn/policy.go

```diff
@@ -33,20 +32,18 @@ func (oc *Controller) syncNetworkPoliciesPortGroup(
 			// policy doesn't exist on k8s. Delete the port group
 			portGroupName := fmt.Sprintf("%s_%s", namespaceName, policyName)
 			hashedLocalPortGroup := hashedPortGroup(portGroupName)
-			oc.deletePortGroup(hashedLocalPortGroup)
+			deletePortGroup(hashedLocalPortGroup)
 
 			// delete the address sets for this policy from OVN
-			oc.deleteAddressSet(hashedAddressSet(addrSetName))
+			deleteAddressSet(hashedAddressSet(addrSetName))
 		}
 	})
 	if err != nil {
 		logrus.Errorf("Error in syncing network policies: %v", err)
 	}
 }
 
-func (oc *Controller) addACLAllow(np *namespacePolicy,
-	match, l4Match string, ipBlockCidr bool, gressNum int,
-	policyType knet.PolicyType) {
+func addACLAllow(np *namespacePolicy, match, l4Match string, ipBlockCidr bool, gressNum int, policyType knet.PolicyType) {
```

Comment:
Do we like long lines now? (I do...)

Reply:
@danwinship I do, as long as they aren't horribly long... I try to keep under like 90 or 100. Worth a try :)